### DISCUSSION TITLE
¿Usamos scanf para el TP2?

### DISCUSSION THREAD

**Discussion by SantiR2212** [INGCOM-UNRN-P1/cursada-2023#83] [Preguntas y respuestas] [answered]
En el TP2 ¿Usamos el `scanf` para pedir los valores de las variables? ¿O no hay que hacerlo por algún motivo en particular?

**Comment by martinvilu** [comment, marked as answer]
Con `scanf`, eventualmente veremos como usar los argumentos de `main`.

(y dicho `scanf`, en el `main` y no en la función a implementar)
